influxdb2_client/src/models/health.rs

```diff
@@ -0,0 +1,52 @@
+//! Health
+
+use serde::{Deserialize, Serialize};
+
+/// HealthCheck
+#[derive(Clone, Debug, PartialEq, Default, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct HealthCheck {
+    /// Name of the influxdb instance
+    pub name: String,
+    /// Message
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub message: Option<String>,
+    /// Checks
+    #[serde(default, skip_serializing_if = "Vec::is_empty")]
+    pub checks: Vec<crate::models::HealthCheck>,
+    /// Status
+    pub status: Status,
+    /// Version
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub version: Option<String>,
+    /// Commit
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub commit: Option<String>,
+}
+
+impl HealthCheck {
+    /// Returns instance of HealthCheck
+    pub fn new(name: String, status: Status) -> Self {
+        Self {
+            name,
+            status,
+            ..Default::default()
```

Comment:
Hm, given that this is currently the only usage of `HealthCheck::default` (I think?), and that `Status` doesn't have a clear default (and `status`'s default doesn't get used in this `new` function anyway), I think in this case it would be better to not have `HealthCheck` or `Status` implement `Default`. That will make this line slightly more annoying, but not by much:

```suggestion
            message: None,
            checks: Vec::new(),
            version: None,
            commit: None,
```

Could you make that change please?